disk/README.md

```diff
@@ -15,7 +15,22 @@ The disk check is included in the [Datadog Agent][1] package, so you don't need
 The Disk check is enabled by default, and the Agent collects metrics on all local partitions. To configure the check with custom options, edit the `disk.d/conf.yaml` file, in the `conf.d/` folder at the root of your [Agent's configuration directory][2]. See the [sample disk.d/conf.yaml][3] for all available configuration options.
 
 #### Note for Windows hosts
-The Agent requires Administrator permissions to collect mount point metrics on hosts running Windows. To collect these metrics without granting Administrator permissions, use the [PDH check][4] to collect mount point metrics from the corresponding perf counters. 
+There are three scenarios where the Disk check can be used:
+
+1. Monitoring physical drives
+
+Monitoring physical drives that are represented by a disk letter (e.g. C:\, D:\ etc...) is supported out of the box by the Disk check without any special considerations.
+
+2. Monitoring nested mount points
+
+Monitoring mounted folders within a filesystem requires Administrator permissions. That's because the underlying Windows function call [FindFirstVolumeMountPoint][9] requires administrative permissions.
```

Comment:
```suggestion
  Monitoring mounted folders within a filesystem requires Administrator permissions. This is because the underlying Windows function call [FindFirstVolumeMountPoint][9] requires administrative permissions.
```